### PR TITLE
Add OpenSSF scorecard workflow

### PR DESCRIPTION
**Pull request by glimchb**
Ensure following best open source security practices by monitoring OpenSSF score of the project.

### CODE REVIEW

File: .github/workflows/scorecard.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+permissions:
+  contents: read
+
+jobs:
+  call:
+    permissions:
+      # Keep in sync with opi-smbios-bridge, no direct way to inherit permissions
+      contents: read
+      security-events: write
```
[the remark this answers is not in the document]
Comment (glimchb) [reply .github/workflows/scorecard.yml:18]: @intelfisz what did you do for intel? dismissed an alert here ?

[the remark this answers is not in the document]
Comment (intelfisz) [reply .github/workflows/scorecard.yml:18]: @glimchb - I treat it as a false positive and assume you can safely dismiss it. For the original workflow in smbios-bridge it does not report it but it does for a reused one. I assume this behavior facilitates double-checking if write permissions are needed for reused 3rd party workflows.

**Review by intelfisz — APPROVED** [review]

File: .github/workflows/scorecard.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (glimchb) [reply .github/workflows/scorecard.yml:18]: @intelfisz do we still need the permissions in this file ? or can be removed ??

[the remark this answers is not in the document]
Comment (intelfisz) [reply .github/workflows/scorecard.yml:18]: you can try removing all and see (possibly accepting some false positives) but as far as I remember it complains about not having top-level policy set and if you set it, then you need to override on job level

[the remark this answers is not in the document]
Comment (glimchb) [reply .github/workflows/scorecard.yml]: trying...

**Review by sandersms — APPROVED** [review]

**Review by artek-koltun — APPROVED** [review]

**Pull request merged**